src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/uat/common/InboundArgumentsProvider.java

```diff
@@ -0,0 +1,43 @@
+package uk.nhs.digital.nhsconnect.lab.results.uat.common;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class InboundArgumentsProvider extends AbstractArgumentsProvider  {
```

Comment:
Is this needed?

Reply:
No it isn't being used anywhere so I've removed it